src/SummerShop.WebApi/Controllers/ProductController.cs

```diff
@@ -0,0 +1,55 @@
+﻿using AutoMapper;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using SummerShop.WebApi.Data;
+using SummerShop.WebApi.Models.Product;
+
+namespace SummerShop.WebApi.Controllers;
+
+[Route("API/[controller]")]
```

Comment:
This is more of a question instead of a review.

Would api path for this specific controll be API/products? or api/products

Reply:
Unless configured, this would be API/products du to case sensitivity. Imo better to do api/[controller] so I will change.
Always room to do some things in program.cs like `services.AddRouting(options => options.LowercaseUrls = true);`